### PR TITLE
Updates to various pulp packages

### PR DESCRIPTION
**Pull request by pcreech**

### CODE REVIEW

**Comment by pcreech** [comment]
Still waiting on confirmation that pulpcore-client's version number

**Comment by pcreech** [comment]
confimed.  pulpcore-client version is correct

File: packages/python-pulp-rpm/python-pulp-rpm.spec [hunk 1]
```diff
@@ -88,6 +88,9 @@ set -ex
 
 
 %changelog
+* Thu May 12 2022 Patrick Creech <pcreech@redhat.com> - 3.17.5-1
```
Comment (Odilhao) [inline packages/python-pulp-rpm/python-pulp-rpm.spec:91]: createrepo_c was bumped from 3.17.3 to 3.17.5

```
createrepo_c~=0.17.6
createrepo_c~=0.20.0
```

https://github.com/pulp/pulp_rpm/compare/3.17.3...3.17.5#diff-4d7c51b1efe9043e44439a949dfd92e5827321b34082903477fd04876edb7552

**Review by Odilhao — APPROVED** [review]

**Comment by pcreech** [comment]
repoclosure expected, as createrepo_c is in koji, but the repo hasn't been mashed yet.

**Comment by Odilhao** [comment]
This repoclosure is expected, the build is tagged into 3.16, but it was not released yet

**Pull request merged**